hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/ContractKey.java

```diff
@@ -133,7 +132,7 @@ public boolean equals(final Object o) {
     /** Special hash to make sure we get good distribution. */
```

Comment:
How about, until the PR which is going to provide a custom-named _persistent_ hash function for `ContractKey` lands, you enhance this javadoc with scary-sounding threats to intimidate anyone away from using this for some _other_ persistent use than the one we're currently suffering from?